noxfile.py

```diff
@@ -131,7 +136,7 @@ def git_clone(session: nox.Session, git_url: str) -> None:
         session.run("git", "-C", expected_directory, "pull", external=True)
 
 
-@nox.session()
+@nox.session(venv_backend="virtualenv")
```

Comment:
Why is this needed for botocore but not requests?

Reply:
botocore job failed with uv